frontend/src/components/Login.js

```diff
@@ -1,126 +1,132 @@
 import React, { useState } from 'react';
-import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { User } from 'react-feather';
 import config from '../config';
+import {
+  FieldFactory,
+  PrimaryButton,
+  Label,
+} from './UI';
 
-const Form = styled.form`
-background-color: #262626;
-width: 400px;
-height: 400px;
-margin: 7em auto;
-border-radius: 1.5em;
-box-shadow: 0px 11px 35px 2px rgba(0, 0, 0, 0.14)
-`;
-const Button = styled.button`
-position: relative;
-margin: 15px 0 5px 0;
-margin-top: 0px;
-margin-left:80px;
-left: 30%;
-transform: translateX(-50%);
-background: #9677D9;
-border: none;
-border-radius: 5px;
-padding: 10px;
-color: white;
-width: 30%;
-font-family: "Noto Sans", sans-serif;
-font-size: 14px;
-cursor: pointer;
-    `;
-const Input = styled.input`
-border: none;
-border-bottom: 1px solid #333;
-background-color: #262626;
-margin-bottom: 60px;
-width: 90%;
-font-size: 18px;
-line-height: 1.2;
-outline: none;
-color: #9677D9;
-::-webkit-input-placeholder { /* Chrome/Opera/Safari */
-color: #9677D9;
-font-size: medium;
-font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
-opacity: 0.3;
-}
+const StyledInput = FieldFactory(styled.input``);
+
+const Icon = styled(User)`
+margin:0 auto
 `;
-const H3 = styled.h3`
-padding-top: 40px;
-color: #8C55AA;
-font-family: 'Ubuntu', sans-serif;
-font-weight: bold;
-font-size: 23px;
-margin-left:150px;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  background-color: ${(props) => props.theme.colors.fill};
+  padding: ${(props) => props.theme.padding.section};
+  margin: 30% auto;
+  width: 90%;
+  max-width: 26rem;
+  border-radius: ${(props) => props.theme.size.corner};
+  -webkit-box-shadow: -10px 10px 40px 0px rgba(10,10,10,0.75);
+  -moz-box-shadow: -10px 10px 40px 0px rgba(10,10,10,0.75);
+  box-shadow: -10px 10px 40px 0px rgba(10,10,10,0.75);
 `;
-const Label = styled.label`
-color: #9677D9;
-font-size: large;
+
+const StyledTextRow = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: ${(props) => props.theme.spacing.subsection};
+  &:nth-child(1) {
+  margin: 0;
+  }
 `;
-const P = styled.p`
-color: #9677D9;
-font-size: large;
+
+const StyledSelectRow = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  margin-top: ${(props) => props.theme.spacing.subsection};
 `;
-const Div = styled.div`
-.form-group{
-margin-top:20px;
-}
+
+const Button = css`
+  padding: ${(props) => props.theme.size.button};
+  width: 100%;
+  margin-top: ${(props) => props.theme.spacing.subsection};
```

Comment:
Kan du flytta dessa rader in till StyledPrimaryButton där under? Och sedan ta bort Button?
Är lättare att läsa koden då 😃